src/entity.ts

```diff
@@ -62,12 +62,9 @@ export class Entity {
 				}
 
 				// Set values of new entity for provided properties
-				if (context.isAsync && isNew && properties)
+				if (isNew && properties)
```

Comment:
@tskimmett Looks like this `isAsync` property can go away now since its not used? Unless you think there's value in keeping it.